src/main/java/com/mathworks/ci/MatlabItemListener.java

```diff
@@ -0,0 +1,91 @@
+package com.mathworks.ci;
+
+/**
+ * Copyright 2020 The MathWorks, Inc.
+ *
+ * Item listener class to provide functionality to check UI element states for a
+ * Multi-configuration project.
+ *
+ */
+
+import hudson.Extension;
+import hudson.matrix.MatrixConfiguration;
+import hudson.matrix.MatrixProject;
+import hudson.model.Item;
+import hudson.model.TopLevelItem;
+import hudson.model.listeners.ItemListener;
+import jenkins.model.Jenkins;
+
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@Extension
+public final class MatlabItemListener extends ItemListener {
+    private static final Map<String, Boolean> prjCheckMATLABAxis = new HashMap<>();
```

Comment:
+1 on @mcafaro  's comments for using MATLAB for customer facing UI and Matlab othrwise

Reply:
Done